### PR TITLE
i18n(es): Update `sharing-state`, `flightcontrol` & `github`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Updated `sharing-state`, `flightcontrol` & `github` based on these commits:
- 9fe3b6fa1594f64b1751c5bb1929bca1a3905a82
- 79d4fee2b23cba9922a5a7cdc4e8dc51d004ac55
- e8b56cdac2f66b6e22d6983c681dd726c9bf7db9

**Note:** the `github` file was very outdated.

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/guides/deploy/github.mdx [hunk 1]
```diff
@@ -86,28 +124,21 @@ Astro mantiene la acción oficial `withastro/action` para desplegar tu proyecto
...
+
     :::note
-    La acción de astro toma algunas entradas opcionales. Estas pueden ser proporcionadas mediante descomentar la línea de `with:` y la entrada que quieras utilizar.
+    La acción de astro toma algunas entradas opcionales. Estas pueden ser proporcionadas descomentando la línea `with:` y la entrada que quieres usar.
     :::
-
+    
     :::caution
-    La [action oficial de Astro](https://github.com/withastro/action) busca un archivo lockfile para detectar tu gestor de paquetes (`npm`, `yarn`, `pnpm` o `bun`). Debes tener el archivo `package-lock.json`, `yarn.lock`, `pnpm-lock.yaml` o `bun.lockb`, generado automáticamente por tu gestor de paquetes, en tu repositorio.
+    La [acción](https://github.com/withastro/action) oficial de Astro escanea el lockfile para detectar tu gestor de paquetes preferido (`npm`, `yarn`, `pnpm`, o `bun`). Debes commitear el archivo `package-lock.json`, `yarn.lock`, `pnpm-lock.yaml` o `bun.lockb` a tu repositorio.
```
Comment (Waxer59) [inline src/content/docs/es/guides/deploy/github.mdx:133]:
```suggestion
    La [acción](https://github.com/withastro/action) oficial de Astro escanea el lockfile para detectar tu gestor de paquetes preferido (`npm`, `yarn`, `pnpm`, o `bun`). Debes incluir el archivo `package-lock.json`, `yarn.lock`, `pnpm-lock.yaml` o `bun.lockb` en tu repositorio.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia!

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 🫡

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Pull request merged**